Week-7/7.1/index7.1.js

```diff
@@ -0,0 +1,47 @@
+class SLLNode {
+    constructor(value) {
+        this.value = value;
+        this.next = null;
+    }
+}
+
+function createSLL(arr) {
+    let head = new SLLNode(arr[0]);
+    let curr = head;
+
+    for(let i=1 ; i<arr.length ; i++) {
+        curr.next = new SLLNode(arr[i]);
+        curr = curr.next;
+    }
+
+    return head;
+}
+
+function traverseSLL(head) {
+    let arr = [];
+
+    let curr = head;
+    while(curr != null) {
+        arr.push(curr.value);
+        curr = curr.next;
+    }
+
+    console.log("SLL: ", arr);
+}
+
+function reverseSLL(head) {
+    let curr = head;
```

Comment:
@mr-rego Can you do this by interchanging next pointers with respect to linked list and not use an array instead